src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,125 @@
+
+## Calibration
+
+To calibrate a servo perform the following instructions:
+
+1. Select "**Tools -> Calibration**". A window will pop up with instructions on how to calibrate the servo.
+
+2. Select the correct servo model. The model will be written on the servo (either MX-64 for head and arm servos or MX-106 for leg servos). Do **NOT** select the 2.0 versions.
+
+3. Select the correct communication port from the dropdown menu.
+
+4. Disconnect the servo from the USB attachment for a few seconds and then reconnect. If the software does not detect the disconnection, retry. If it fails, redo the calibration from step one. The software will then install the firmware to the servo.
+
+5. Rotate the servo so that the dot is in the 6-o'clock position (as seen in the instructions). Use the tool pictured to move the horn and use the marks on the tool to line up the dot correctly.
+
+   ![Photo of the servo rotating tool](./images/servo_calibration/servo_tool.jpg)
+
+6. Continue to rotate to each of the 3-o'clock, 12-o'clock and 9-o'clock positions when told by the software.
+
+## Parameters
+
+To update the servo parameters, perform the following instructions:
+
+1. Select "**Scan**" and wait for the attached servo to be discovered.
+
+   Optional: brand new servos should be preset to an ID of 1 and Baud Rate of 57600 bps. Lower the ID and Baud Rate range in "**Options > Scan**" to speed up discovery.
+   ![Screenshot of Dynamixel Wizard performing a scan with no servos discovered](./images/servo_calibration/scan.png)
+   ![Screenshot of Dynamixel Wizard performing a scan with 3 servos discovered](./images/servo_calibration/discover.png)
+
+2. If there are multiple servos discovered, select the option with ID = 1 and Baud Rate = 57600 bps.
+
+3. Update the following parameters (don't forget to press save after each update):
+
+   - ID: Varies depending on where the servo will be placed. See below for guidance.
+
+     | Servo Position       | ID  |
+     | -------------------- | --- |
+     | Right Shoulder Pitch | 1   |
+     | Left Shoulder Pitch  | 2   |
+     | Right Shoulder Roll  | 3   |
+     | Left Shoulder Roll   | 4   |
+     | Right Elbow          | 5   |
+     | Left Elbow           | 6   |
+     | Right Hip Yaw        | 7   |
+     | Left Hip Yaw         | 8   |
+     | Right Hip Roll       | 9   |
+     | Left Hip Roll        | 10  |
+     | Right Hip Pitch      | 11  |
+     | Left Hip Pitch       | 12  |
+     | Right Knee           | 13  |
+     | Left Knee            | 14  |
+     | Right Ankle Pitch    | 15  |
+     | Left Ankle Pitch     | 16  |
+     | Right Ankle Roll     | 17  |
+     | Left Ankle Roll      | 18  |
+     | Head Yaw             | 19  |
+     | Head Pitch           | 20  |
+
+     Note that roll is about the x-axis, pitch is about the y-axis, and yaw is about the z-axis. The below diagram specifies the position of each servo within the robot when it is in its default (zombie) position.
+     <details>
```

Comment:
Won't this have the same indentation issue?

Reply:
It seems to work fine for me?